Tools/TestWebKitAPI/Tests/WebKitCocoa/LockdownModeFonts.mm

```diff
@@ -205,11 +205,10 @@
 
 #if PLATFORM(WATCHOS)
         EXPECT_NE(targetResult, referenceResult);
+#elif HAVE(CTFONTMANAGER_CREATEMEMORYSAFEFONTDESCRIPTORFROMDATA)
+        EXPECT_NE(targetResult, referenceResult);
 #else
-    if (!PAL::canLoad_CoreText_CTFontManagerCreateMemorySafeFontDescriptorFromData())
         EXPECT_EQ(targetResult, referenceResult);
-    else
-        EXPECT_NE(targetResult, referenceResult);
```

Comment:
Maybe?

```cpp
#if PLATFORM(WATCHOS) || HAVE(CTFONTMANAGER_CREATEMEMORYSAFEFONTDESCRIPTORFROMDATA)
        EXPECT_NE(targetResult, referenceResult);
#else
        EXPECT_EQ(targetResult, referenceResult);
#endif
```

Reply:
Now that I think about it, `HAVE(CTFONTMANAGER_CREATEMEMORYSAFEFONTDESCRIPTORFROMDATA)` will be true for all watchOS versions supported on main, so you can just:

```cpp
#if HAVE(CTFONTMANAGER_CREATEMEMORYSAFEFONTDESCRIPTORFROMDATA)
        EXPECT_NE(targetResult, referenceResult);
#else
        EXPECT_EQ(targetResult, referenceResult);
#endif
```

Reply:
Good point!